docs/guides/src/main/server/logging.adoc

```diff
@@ -116,118 +130,130 @@ By default, the console log handler logs plain unstructured data to the console.
 
 === Enable file logging
-Logging to a file is disabled by default. To enable it, run the following command:
+Logging to a file is disabled by default. To enable it, enter the following command:
 
 <@kc.start parameters="--log=console,file"/>
 
-=== Configuring path and name of the generated log file
-By enabling the file log handler, a log file named `keycloak.log` will be created inside the `data/log` directory of your Keycloak installation.
+A log file named `keycloak.log` is created inside the `data/log` directory of your Keycloak installation.
 
-To change the location and name of the generated log file, run the following command:
+=== Configuring the location and name of the log file
 
-<@kc.start parameters="--log=console,file --log-file=<path-to>/<your-file.log>"/>
+To change where the log file is created and the file name, perform these steps:
 
-Please make sure the location for the logfile is writeable. If not, an error will be thrown at start-up. Keycloak will start correctly, but no file containing logs will be created.
+. Create a writable directory to store the log file.
++
+If the directory is not writable, Keycloak will start correctly, but it will issue an error and no log file will be created.
+
+. Enter this command:
++
+<@kc.start parameters="--log=console,file --log-file=<path-to>/<your-file.log>"/>
 
 === Configuring the file handler format
-You can configure a different logging format for the file log handler by running the following command:
+To configure a different logging format for the file log handler, enter the following command:
 
 <@kc.start parameters="--log-file-format=<pattern>"/>
 
 Please see the <<Configuring the console log format>> section in this guide for more information and a table of the available pattern configuration.
 
-== Centralized logging using GELF
-Keycloak is able to send logs to a centralized log management system like Graylog, Logstash (inside the Elastic Stack or ELK - Elasticsearch, Logstash, Kibana) or Fluentd (inside EFK - Elasticsearch, Fluentd, Kibana). Keycloak leverages the features of the https://quarkus.io/guides/centralized-log-management[Quarkus Logging GELF] extension to provide support for these environments.
+== Centralized logging using Gelf
+Keycloak can send logs to a centralized log management system such as the following:
+
+* Graylog
+* Logstash, inside the Elasticsearch, Logstash, Kibana (ELK) logging stack
+* Fluentd, inside the Elasticsearch, Fluentd, Kibana (EFK) logging stack
 
-=== Enable the GELF handler
-To enable logging using GELF, you have to add it to the list of activated log handlers.
+Keycloak uses the https://quarkus.io/guides/centralized-log-management[Quarkus Logging Gelf] extension to support these environments.
+
+=== Enabling the Gelf handler
+To enable logging using Gelf, add it to the list of activated log handlers.
 
 .Example:
 <@kc.start parameters="--log=console,gelf"/>
 
-=== Configure the GELF handler
+=== Configuring the Gelf handler
```

Comment:
I changed this to use GELF on purpose as part of #13422, looking at the actual https://docs.graylog.org/docs/gelf docs and others. It is an acronym for Graylog Extended Log Format. Pls change back to `GELF` everywhere, so we are consistent with other original sources and our CLI.